Changes.md

```diff
@@ -105,8 +105,9 @@ API
   - Added `scrollToNextMessage()` and `scrollToPreviousMessage()` methods.
 - MetadataAlgo : Added `readOnlyReason`, returning the outer-most `GraphComponent` that causes the specified component to be read-only.
 - EditScopeAlgo : Added `prunedReadOnlyReason`, `transformEditReadOnlyReason` and `parameterEditReadOnlyReason` to determine the outer-most `GraphComponent` causing and edit (or potential edit) to be read-only.
-- Window : Changed `setPosition` such that the whole window will remain on screen. Set `forcePosition` to `False` to disable this behaviour.
+ - Window : Changed `setPosition` such that the whole window will remain on screen. Set `forcePosition` to `False` to disable this behaviour.
```

Comment:
Is this indentation accidental?

Reply:
Sorry, no I wasn't. I think it crept in whilst fixing a conflict and I accidentally had whitespace diff off. Fixed in the squash.